### PR TITLE
Remove debug print statements

### LINKED ISSUE #714: Bug: `fileNameFlat` is not namelist configurable.
`fileNameFlat` is namelist configurable as seen from line [154](https://github.com/NOAA-EMC/UPP/blob/c450a49c2afbabd5b5e126e90b8ec07d7099a669/sorc/ncep_post.fd/WRFPOST.f#L154).
This variable is read from the namelist `itag` on line [201](https://github.com/NOAA-EMC/UPP/blob/c450a49c2afbabd5b5e126e90b8ec07d7099a669/sorc/ncep_post.fd/WRFPOST.f#L201) as part of `&model_inputs`.

However, this variable `fileNameFlat` is over-written on line [279](https://github.com/NOAA-EMC/UPP/blob/c450a49c2afbabd5b5e126e90b8ec07d7099a669/sorc/ncep_post.fd/WRFPOST.f#L279).  Hence `fileNameFlat` is not user configurable via the namelist.

The line [201](https://github.com/NOAA-EMC/UPP/blob/c450a49c2afbabd5b5e126e90b8ec07d7099a669/sorc/ncep_post.fd/WRFPOST.f#L201) should be set before the namelist is read, such as at line [197](https://github.com/NOAA-EMC/UPP/blob/c450a49c2afbabd5b5e126e90b8ec07d7099a669/sorc/ncep_post.fd/WRFPOST.f#L197).

A bugfix patch is attached [here](https://github.com/NOAA-EMC/UPP/files/11567790/bugfix.diff.txt).

```diff
 ❯❯❯ git diff
diff --git i/sorc/ncep_post.fd/WRFPOST.f w/sorc/ncep_post.fd/WRFPOST.f
index 2df8a11b..c32bd4d2 100644
--- i/sorc/ncep_post.fd/WRFPOST.f
+++ w/sorc/ncep_post.fd/WRFPOST.f
@@ -195,6 +195,8 @@
 !KaYee: Read itag in Fortran Namelist format
 !Set default
        SUBMODELNAME='NONE'
+!set control file name
+       fileNameFlat='postxconfig-NT.txt'
        numx=1
 !open namelist
        open(5,file='itag')
@@ -212,6 +214,8 @@
          if (me==0) print*,'DateStr= ',DateStr
          if (me==0) print*,'MODELNAME= ',MODELNAME
          if (me==0) print*,'SUBMODELNAME= ',SUBMODELNAME
+         if (me==0) print*,'fileNameFlux= ',fileNameFlux
+         if (me==0) print*,'fileNameFlat= ',fileNameFlat
           if (me==0) print*,'numx= ',numx
 !       if(MODELNAME == 'NMM')then
 !        read(5,1114) VTIMEUNITS
@@ -275,8 +279,6 @@
         rdaod       = .false.
         d2d_chem     = .false.

-!set control file name
-        fileNameFlat='postxconfig-NT.txt'
         read(5,nampgb,iostat=iret,end=119)
  119    continue
        if (me==0) print*,'in itag, write_ifi_debug_files=', write_ifi_debug_files
```

### PR DESCRIPTION
**Pull request by DusanJovic-NOAA**
Reduce the amount of data being printed to stdout by removing debug print statements. Related to https://github.com/ufs-community/ufs-weather-model/issues/1738

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@DusanJovic-NOAA Please sync your branch with the latest UPP develop.

**Comment by DusanJovic-NOAA** [comment]
> @DusanJovic-NOAA Please sync your branch with the latest UPP develop.

Done

File: sorc/ncep_post.fd/WRFPOST.f [hunk 1]
```diff
@@ -199,20 +197,11 @@ PROGRAM WRFPOST
...
        stop
        endif
        
-       if (me==0) print*,'fileName= ',fileName
-         if (me==0) print*,'IOFORM= ',IOFORM
-         !if (me==0) print*,'OUTFORM= ',grib
-         if (me==0) print*,'OUTFORM= ',grib
-         if (me==0) print*,'DateStr= ',DateStr
-         if (me==0) print*,'MODELNAME= ',MODELNAME
-         if (me==0) print*,'SUBMODELNAME= ',SUBMODELNAME
```
Comment (aerorahul) [inline sorc/ncep_post.fd/WRFPOST.f:214]: I agree with this change, however, it is sometimes important to see what the namelist is being set in the Fortran code.
I would recommend replacing the highlighted section with:
```fortran
if (me == 0) write(6, model_inputs)
```
Similarly at line 282.
```fortran
if (me == 0) write(6, nampgb)
```

File: sorc/ncep_post.fd/PROCESS.f [hunk 2]
```diff
@@ -54,79 +54,66 @@ SUBROUTINE PROCESS(kth,kpv,th,pv,iostatusD3D)
 !     START SUBROUTINE PROCESS.
 !
       cfld=0
-      if(me==0) write(*,*) "PROCESS starts"
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/PROCESS.f]: @DusanJovic-NOAA I would like to keep the prints in this subroutine for tracking the process status. They are fewer than 15 lines.

Comment (DusanJovic-NOAA) [reply sorc/ncep_post.fd/PROCESS.f]: Reverted in [8d3255a](https://github.com/NOAA-EMC/UPP/pull/705/commits/8d3255a605430769debbfa6e76ecfa6899171f44)

File: sorc/ncep_post.fd/WRFPOST.f (hunk 1 above)
Comment (DusanJovic-NOAA) [reply sorc/ncep_post.fd/WRFPOST.f:214]: Done in  [dccad7f](https://github.com/NOAA-EMC/UPP/pull/705/commits/dccad7fe92592fbb2814aa82e5e4edc6fd0cfef2)

**Comment by WenMeng-NOAA** [comment]
@FernandoAndrade-NOAA Please conduct the UPP RT tests on Hera and Orion.

**Comment by FernandoAndrade-NOAA** [comment]
@WenMeng-NOAA UPP RTs have finished on Hera and Orion with no changes to results.

**Comment by WenMeng-NOAA** [comment]
> @WenMeng-NOAA UPP RTs have finished on Hera and Orion with no changes to results.

@FernandoAndrade-NOAA Please go head to approve this PR.

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on WCOSS2 without changed results.

**Review by WenMeng-NOAA — APPROVED** [review]

**Review by FernandoAndrade-NOAA — APPROVED** [review]

**Pull request merged**